app/views/hq/users/show.html.erb

```diff
@@ -10,7 +10,8 @@
   <%= pluralize(@user.active_sponsors.count.to_s, 'Active Sponsor') %>
   <% if @user.sponsors.count > 0 %>
```

Comment:
The tests that were written with this will allow either one, this is a style thing